tools/ci/commands.json

```diff
@@ -27,7 +27,7 @@
     "help": "Update built tests",
     "virtualenv": true,
     "requirements": [
-        "requirements_build.txt"
+      "requirements.txt"
```

Comment:
I think we should split out the requirements to build the tests to the requirements that are specific to running taskcluster.